apps/gateway/src/index.ts

```diff
@@ -24,8 +24,16 @@ app.route(jetPaths(ROUTES).Api.V1.Documents.Base, documentsRouter)
 // Must be the last one!
 app.use(errorHandlerMiddleware())
 
-serve(app, (info) => {
-  console.log(`Listening on http://localhost:${info.port}`)
-})
+serve(
+  {
+    fetch: app.fetch,
+    overrideGlobalObjects: undefined,
+    port: parseInt(process.env.GATEWAY_PORT || '4000', 10),
```

Comment:
I had nextjs app running on 3000 and running test of gateway conflicted

Reply:
can't be default gateway port is 8787